velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -48,43 +48,59 @@ static std::vector<std::string> kSortOrderBasedOverClauses = {
     "order by c1 desc nulls first, c0 asc nulls first",
 };
 
-/// Common set of window function over clauses for window frame tests to ensure
-/// total ordering for deterministic results in ROW mode.
-static std::vector<std::string> kFrameOverClauses = {
-    "partition by c0 order by c1, c2",
-    "partition by c1 order by c0, c2",
+/// Common set of window function over clauses using a combination of three
+/// columns, that produces a deterministic, total ordering of all rows.
+static std::vector<std::string> kOrderedBasicOverClauses = {
```

Comment:
We can change the name to kBasicOverClauses, and removed Ordered. That usually is needed for testing to be deterministic and can be understood.

Reply:
Thanks for the suggestion, replaced the original kBasicOverClauses and others with the ordered overClause lists.